src/wallet/rpcwallet.cpp

```diff
@@ -1607,7 +1604,8 @@ static UniValue listsinceblock(const JSONRPCRequest& request)
 
     bool include_removed = (request.params[3].isNull() || request.params[3].get_bool());
 
-    int depth = pindex ? (1 + chainActive.Height() - pindex->nHeight) : -1;
+    const Optional<int> tip_height = locked_chain->getHeight();
+    int depth = tip_height && height ? (1 + *tip_height - *height) : -1;
```

Comment:
in commit 22abf4a Add findFork and findBlock to the Chain interface

Same (I'd prefer to not check for height again, since that has already been done above as you moved the FindFork to before the height check.)

Reply:
Right, my bad.